cli/core/pkg/pluginmanager/manager.go

```diff
@@ -171,32 +223,67 @@ func AvailablePluginsFromLocalSource(localPath string) ([]plugin.Discovered, err
 	if err != nil {
 		log.Warningf("Unable to discover standalone plugins from local source, %v", err.Error())
 	}
-	return availablePlugins("", []plugin.Discovered{}, localStandalonePlugins)
+	return availablePlugins([]plugin.Discovered{}, localStandalonePlugins)
```

Comment:
Is it ok that we no longer specify an empty server here?  IIUC before we used to request only the standalone plugins since the server passed was empty, but now what will happen since we can't specify a server name anymore?